src/libs/events.test.js

```diff
@@ -0,0 +1,59 @@
+import { extractCrossWorkspaceDetails, extractWorkspaceDetails } from 'src/libs/events'
+
+
+const gcpWorkspace = {
+  workspace: {
+    cloudPlatform: 'Gcp',
+    name: 'wsName',
+    namespace: 'wsNamespace',
+    workspaceId: 'testGoogleWorkspaceId'
```

Comment:
Perhaps use the null uuid to document it's always a uuid (likewise with azure below)

Reply:
For the unit tests we don't try to make the data values particularly realistic-- the goal is to test the logic of the methods and make the tests easy to read. The JavaScript methods don't care what the format of the ID is.

Reply:
Arguably, I could (perhaps should?) omit the workspaceIds entirely from this test.